File: JHBaseDemo/ViewController.m
//
//  ViewController.m
//  JHBaseDemo
//
//  Created by lifei on 2021/8/25.
//

#import "ViewController.h"
#import <JHBaseDemo-Swift.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    self.view.backgroundColor = [UIColor lightGrayColor];
}

- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event {
    self.view.backgroundColor =  [UIColor colorWithRed:((float)arc4random_uniform(256) / 255.0) green:((float)arc4random_uniform(256) / 255.0) blue:((float)arc4random_uniform(256) / 255.0) alpha:1.0];
    MainModel *mmm = [[MainModel alloc] init];
    [mmm test];
}


@end

File: JHBaseDemo/JHBaseDemo/ViewController.swift
//
//  ViewController.swift
//  JHBaseDemo
//
//  Created by lifei on 2021/8/27.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        self.view.backgroundColor = .lightGray
    }
    
    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
        testVC()
    }
    
    func testVC() {
        let testVC = JHTestVC.init(title: "测试")
        testVC.modalPresentationStyle = .fullScreen
        present(testVC, animated: true, completion: nil)
    }

}

